examples/mq/v1alpha1/user.yaml

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: mq.aws.upbound.io/v1alpha1
+kind: User
+metadata:
```

Comment:
Let's also add the manual intervention annotation till we consume https://github.com/crossplane/upjet/pull/406 in this provider.